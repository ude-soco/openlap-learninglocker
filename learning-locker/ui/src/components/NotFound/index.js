import React from 'react';

export default function NotFound() {
  return (
    <div className="container">
      <h1>Coming soon!</h1>
      <p>This page is still being built</p>
    </div>
  );
}
