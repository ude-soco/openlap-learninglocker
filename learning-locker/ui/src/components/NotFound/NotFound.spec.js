import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import NotFound from './index';

test('renders correctly with no props', () => {
  const tree = ReactTestRenderer.create(<NotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});
