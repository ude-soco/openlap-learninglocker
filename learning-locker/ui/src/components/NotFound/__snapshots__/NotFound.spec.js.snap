// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with no props 1`] = `
<div
  className="container"
>
  <h1>
    Coming soon!
  </h1>
  <p>
    This page is still being built
  </p>
</div>
`;
